src/checkoutservice/main.go

```diff
@@ -98,6 +99,28 @@ func main() {
 	log.Fatal(err)
 }
 
+func initJaegerTracing() {
+	svcAddr := os.Getenv("JAEGER_SERVICE_ADDR")
+	if svcAddr == "" {
+		log.Info("jaeger initialization disabled.")
+		return
+	}
+
+	// Register the Jaeger exporter to be able to retrieve
+	// the collected spans.
+	exporter, err := jaeger.NewExporter(jaeger.Options{
+		Endpoint: fmt.Sprintf("http://%s", svcAddr),
+		Process: jaeger.Process{
+			ServiceName: "checkoutservice",
```

Comment:
Oh I wonder how does stackdriver exporter automatically know the name of the service (I don't think we pass it below?) but jaeger doesn't.

No action needed for this, just chatting.